src/program.js

```diff
@@ -246,6 +246,14 @@ Example: $0 --help run.
       describe: 'Show verbose output',
       type: 'boolean',
     },
+    'ignore-files': {
```

Comment:
It seems that setting `requiresArg` would force yargs to validate `ignoreFiles` is not a empty list. Since the `ignoreFiles` defaults to an empty list, this make yargs always argue `Missing argument value: ignore-files`.